compiler/record-minmax/src/HDF5Importer.h

```diff
@@ -39,9 +39,12 @@ namespace record_minmax
 class HDF5Importer
 {
 public:
-  explicit HDF5Importer(const std::string &path) : _file{path, H5F_ACC_RDONLY}
+  explicit HDF5Importer(const std::string &path)
   {
-    // Do nothing
+    if (_file.isHdf5(path) == false)
+      throw std::runtime_error("Given data file is not HDF5");
```

Comment:
(optional) "Error: Given data file is not HDF5" ?

Reply:
Master branch is already merged with same error message :(
How about applying it from 1.10.0?

Reply:
We can apply another change to master :)